starlette/testclient.py

```diff
@@ -365,6 +380,14 @@ def receive_json(self, mode: str = "text") -> typing.Any:
 class TestClient(requests.Session):
     __test__ = False  # For pytest to not discover this up.
 
+    #: These options are passed to `anyio.start_blocking_portal()`
+    async_backend = {
+        "backend": "asyncio",
+        "backend_options": {},
+    }  # type: typing.Dict[str, typing.Any]
```

Comment:
Nowhere else in this file was variable annotation being used, but I'll change this one since starlette depends on python 3.6+

Reply:
@JayH5 this is fixed

Reply:
Yeah I know it's a bit inconsistent in the code base, just don't want to add more comment-style annotations.

Reply:
@JayH5 @tomchristie I opened https://github.com/encode/starlette/pull/1189 to fix annotations across the whole project.  If this could be merged soon-ish then the workflows for this PR could be run without approval.